pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -36,6 +39,7 @@ func (r *RegistryFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&r.Anon, "registry-anon", false, "Set anonymous auth ($IMGPKG_ANON)")
 
 	cmd.Flags().DurationVar(&r.ResponseHeaderTimeout, "registry-response-header-timeout", 30*time.Second, "Maximum time to allow a request to wait for a server's response headers from the registry (ms|s|m|h)")
+	cmd.Flags().IntVar(&r.NumberRetries, "registry-num-retries", 5, "Set the number of times imgpkg retries to send requests to the registry in case of an error")
```

Comment:
may be --registry-retry-num/count? would be nice to group retry settings under --registry-retry-*

Reply:
Fixed it